packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,191 @@
+@breakpoints {
+  .rt-DataListRoot.rt-r-gap-0 {
+    row-gap: 0px;
+    column-gap: 0px;
+  }
+  .rt-DataListRoot.rt-r-gap-1 {
+    row-gap: var(--space-1);
+    column-gap: var(--space-1);
+  }
+  .rt-DataListRoot.rt-r-gap-2 {
+    row-gap: var(--space-2);
+    column-gap: var(--space-2);
+  }
+  .rt-DataListRoot.rt-r-gap-3 {
+    row-gap: var(--space-3);
+    column-gap: var(--space-3);
+  }
+  .rt-DataListRoot.rt-r-gap-4 {
+    row-gap: var(--space-4);
+    column-gap: var(--space-4);
+  }
+  .rt-DataListRoot.rt-r-gap-5 {
+    row-gap: var(--space-5);
+    column-gap: var(--space-5);
+  }
+  .rt-DataListRoot.rt-r-gap-6 {
+    row-gap: var(--space-6);
+    column-gap: var(--space-6);
+  }
+  .rt-DataListRoot.rt-r-gap-7 {
+    row-gap: var(--space-7);
+    column-gap: var(--space-7);
+  }
+  .rt-DataListRoot.rt-r-gap-8 {
+    row-gap: var(--space-8);
+    column-gap: var(--space-8);
+  }
+  .rt-DataListRoot.rt-r-gap-9 {
+    row-gap: var(--space-9);
+    column-gap: var(--space-9);
+  }
+
+  .rt-DataListRoot.rt-r-gap-x-0 {
+    row-gap: 0px;
+  }
+  .rt-DataListRoot.rt-r-gap-x-1 {
+    row-gap: var(--space-1);
+  }
+  .rt-DataListRoot.rt-r-gap-x-2 {
+    row-gap: var(--space-2);
+  }
+  .rt-DataListRoot.rt-r-gap-x-3 {
+    row-gap: var(--space-3);
+  }
+  .rt-DataListRoot.rt-r-gap-x-4 {
+    row-gap: var(--space-4);
+  }
+  .rt-DataListRoot.rt-r-gap-x-5 {
+    row-gap: var(--space-5);
+  }
+  .rt-DataListRoot.rt-r-gap-x-6 {
+    row-gap: var(--space-6);
+  }
+  .rt-DataListRoot.rt-r-gap-x-7 {
+    row-gap: var(--space-7);
+  }
+  .rt-DataListRoot.rt-r-gap-x-8 {
+    row-gap: var(--space-8);
+  }
+  .rt-DataListRoot.rt-r-gap-x-9 {
+    row-gap: var(--space-9);
+  }
+
+  .rt-DataListRoot.rt-r-gap-y-0 {
+    column-gap: 0px;
+  }
+  .rt-DataListRoot.rt-r-gap-y-1 {
+    column-gap: var(--space-1);
+  }
+  .rt-DataListRoot.rt-r-gap-y-2 {
+    column-gap: var(--space-2);
+  }
+  .rt-DataListRoot.rt-r-gap-y-3 {
+    column-gap: var(--space-3);
+  }
+  .rt-DataListRoot.rt-r-gap-y-4 {
+    column-gap: var(--space-4);
+  }
+  .rt-DataListRoot.rt-r-gap-y-5 {
+    column-gap: var(--space-5);
+  }
+  .rt-DataListRoot.rt-r-gap-y-6 {
+    column-gap: var(--space-6);
+  }
+  .rt-DataListRoot.rt-r-gap-y-7 {
+    column-gap: var(--space-7);
+  }
+  .rt-DataListRoot.rt-r-gap-y-8 {
+    column-gap: var(--space-8);
+  }
+  .rt-DataListRoot.rt-r-gap-y-9 {
+    column-gap: var(--space-9);
+  }
+}
+
+.rt-DataListRoot {
+  margin: 0;
+  display: grid;
+  /** Value  set via props in component **/
+  grid-template-columns: var(--data-list-columns, minmax(200px, max-content) 1fr);
```

Comment:
I chose this default `minmax(200px, max-content) 1fr)` so the label column width will extend automatically even with really long labels. See this example below that doesn't require a custom label item width. 
<img width="568" alt="Screenshot 2024-01-16 at 3 55 37 PM" src="https://github.com/radix-ui/themes/assets/36613477/9b64eb76-88bc-439e-9376-41b76170a038">
<img width="622" alt="Screenshot 2024-01-16 at 3 55 58 PM" src="https://github.com/radix-ui/themes/assets/36613477/33d0d64d-02a8-486f-acaa-98a858664f66">